lobsterpy/cohp/test/test_describe.py

```diff
@@ -400,78 +439,147 @@ def test_text(self):
+                "from the Ti(3dz2)-O(2pz) orbital, contributing 14.0 percent, whereas the maximum "
+                "antibonding contribution is from the Ti(4s)-O(2s) orbital, contributing 20.0 percent.",
+            ],
+        )
+        self.assertEqual(
+            self.describe_C_orb.text,
+            [
+                "The compound C has 1 symmetry-independent atoms(s) with relevant bonds: C1.",
+                "C1 has a tetrahedral (CN=4) coordination environment. "
+                "It has 4 C-C (mean ICOHP: -9.59 eV, 0.0 percent antibonding interaction below EFermi) bonds.",
+                "In the 4 C-C bonds, relative to the summed ICOHPs, the maximum bonding contribution is "
+                "from the C(2s)-C(2s) orbital, contributing 10.0 percent, whereas no significant "
+                "antibonding contribution is found in this bond.",
+            ],
+        )
+        self.assertEqual(
+            self.describe_NaSbF6_orb.text,
+            [
+                "The compound NaSbF6 has 3 symmetry-independent atoms(s) with relevant bonds: Na1, Sb2, F3.",
+                "Na1 has an octahedral (CN=6) coordination environment. It has 6 Na-F (mean ICOHP: -0.61 eV, "
+                "4.071 percent antibonding interaction below EFermi) bonds.",
+                "In the 6 Na-F bonds, relative to the summed ICOHPs, the maximum bonding contribution is "
+                "from the Na(3s)-F(2s) orbital, contributing 67.0 percent, whereas the maximum antibonding "
+                "contribution are from Na(2py)-F(2s), Na(2pz)-F(2s), and Na(2px)-F(2s) orbitals, "
+                "contributing 13.0, 13.0, and 13.0 percent, respectively.",
+                "Sb2 has an octahedral (CN=6) coordination environment. It has 6 Sb-F (mean ICOHP: -5.45 eV, "
+                "0.0 percent antibonding interaction below EFermi) bonds.",
+                "In the 6 Sb-F bonds, relative to the summed ICOHPs, the maximum bonding contribution "
+                "is from Sb(5py)-F(2s), Sb(5pz)-F(2s), and Sb(5px)-F(2s) orbitals, contributing 14.0, "
+                "14.0, and 14.0 percent, respectively, whereas no significant antibonding contribution is "
+                "found in this bond.",
+                "F3 has a linear (CN=2) coordination environment. It has 1 F-Na (mean ICOHP: -0.61 eV, "
+                "4.545 percent antibonding interaction below EFermi), and 1 F-Sb (mean ICOHP: -5.45 eV, "
+                "0.0 percent antibonding interaction below EFermi) bonds.",
+                "In the 1 F-Na bond, relative to the summed ICOHPs, the maximum bonding contribution "
+                "is from the F(2s)-Na(3s) orbital, contributing 68.0 percent, whereas the maximum "
+                "antibonding contribution are from F(2s)-Na(2pz) and F(2pz)-Na(2pz) orbitals, "
+                "contributing 36.0 and 36.0 percent, respectively.In the 1 F-Sb bond, relative "
```

Comment:
```suggestion
                "contributing 36.0 and 36.0 percent, respectively. In the 1 F-Sb bond, relative "
```
Could you add a space between the sentences, please?

Reply:
Done